index.bs

```diff
@@ -790,6 +790,11 @@ spec:css-syntax-3;
         interfaces it refers to=] have overridden {{Credential/isConditionalMediationAvailable()}} to return
         a new {{Promise}} that [=resolves=] with `true`.
 
+        For {CredentialsContainer/create()}, if a user has previously consented to credential creation and
+        the user agent knows it recently mediated an authentication, then the `create()` call may resolve without
+        additional prominent modal interaction. If the user agent did not recently mediate an authentication or
+        does not have consent for credential creation, then the call must throw a "{{NotAllowedError}}" {{DOMException}}.
```

Comment:
> the call must throw a "{{NotAllowedError}}" {{DOMException}}.

We should spell this out on the `Create a credential` algorithm below.

Reply:
Maybe this should actually say, "will return [=a promise rejected with=] a "{{NotAllowedError}}" {{DOMException}}", as the method itself doesn't "throw", it returns a promise.